application/extensions/AlertWidget/views/alert.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/** @var String $tag */
+/** @var String $text */
+/** @var String $header */
+/** @var boolean $showIcon */
+/** @var boolean $showCloseButton */
+/** @var array $errors */
+/** @var boolean $inErrorMode */
+/** @var array $htmlOptions */
+/** @var String $icon */
+
+echo CHtml::openTag($tag, $htmlOptions);
```

Comment:
Hm do we use CHtml?

Reply:
Allover LS, and it is very good, when you need to pass $htmlOptions